### PR TITLE
Merging changes from master into next_q3_16 branch

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Merging in master to pick up bug fix.

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [x] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,6 @@ python:
 install: "pip install -r requirements-dev.txt"
 # command to run tests
 script: 
-  - nosetests tests.unit_tests
+  - nosetests tests
```
Comment (sagesmith-wf) [inline .travis.yml]: Nice!

**Comment by sagesmith-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
QA +10 - master merge.  Merging.

**Pull request merged**